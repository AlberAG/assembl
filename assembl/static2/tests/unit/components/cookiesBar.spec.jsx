import React from 'react';
import renderer from 'react-test-renderer';

import CookiesBar from '../../../js/app/components/cookiesBar';

describe('CookiesBar component', () => {
  it('should render a bar with 2 buttons to accept the cookies policy or read them', () => {
    const component = renderer.create(<CookiesBar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});