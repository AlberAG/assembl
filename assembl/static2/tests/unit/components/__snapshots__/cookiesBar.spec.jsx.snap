// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookiesBar component should render a bar with 2 buttons to accept the cookies policy or read them 1`] = `
<div
  className="cookies-bar show-cookies-bar"
>
  <span
    className="cookies-text"
    style={undefined}
  >
    Assembl uses cookies to offer you the best possible experience. To use Assembl, you have to agree to the cookies policy of the debate.
  </span>
  <div
    className="cookies-buttons-container"
  >
    <button
      className="button-submit button-dark cookies-button btn btn-default"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      <span
        className={undefined}
        style={undefined}
      >
        I agree
      </span>
    </button>
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <button
        className="button-submit button-dark cookies-button btn btn-default"
        disabled={false}
        type="button"
      >
        <span
          className={undefined}
          style={undefined}
        >
          See the cookies policy
        </span>
      </button>
    </a>
  </div>
</div>
`;
